### ISSUE TITLE
Data Cleaning/Creation

### ISSUE THREAD

**Issue by WillLangas** [WillLangas/CityClassifier#1]
Create a mid range dataset (3K-5K images per city), maybe 10 or so cities

No need to create a separate validation set, you can do this automatically with Kerala

Folder structure:
Data
->train
--->city1
--->city2
...

For the actual images, they need to be resized to 224 x 224. You can use the image.preprocessing module from Keras to do this automatically.

Upload the Zip of it to google drive and share with group
